roles/test_report_send/tasks/reporting/splunk.yml

```
@@ -0,0 +1,126 @@
+  when:
+    - ansible_verbosity | default(0) > 1
+
+
+# data conversion copied from @cplacani PR: https://github.com/openshift-kni/eco-validation/pull/150/files
+
+- name: Combine additional attributes into the data at trs_collector_target for {{ trs_collector }}
+  ansible.builtin.set_fact:
+    trs_event_payload:
+      "source": "{{ trs_collector_source }}"
+      "event": "{{ trs_event }}"
+-     "host": "{{ trs_collector_url }}"
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)